src/components/Banner.tsx

```diff
@@ -1,39 +1,40 @@
-import React from 'react'
-import Button from './button/Button'
+import React from 'react';
+import Button from './button/Button';
 import { PageTitle, PageSubTitle, Title, Desc } from "./text/Text";
-import Onboard from 'bnc-onboard'
-import Web3 from 'web3'
+import Onboard from 'bnc-onboard';
+import Web3 from 'web3';
 import { setUserSession } from '../store/localstorage';
 
 let web3
 const onboard = Onboard({
-    dappId: 'b4ae356c-11d5-4439-baf0-8f452f2bdbcd',  // [String] The API key of Blocknative
-    networkId: 3,  // [Integer] The Ethereum network ID your Dapp uses.
-    subscriptions: {
-      wallet: wallet => {
-         web3 = new Web3(wallet.provider)
-      }
+  dappId: 'b4ae356c-11d5-4439-baf0-8f452f2bdbcd',  // [String] The API key of Blocknative
```

Comment:
Please put sensitive information in .env file and fetch it from there to update dappId, so that its not pushed onto the repository.